software/Nightmare_9dof_imu/Nightmare_9dof_imu.ino

```diff
@@ -0,0 +1,44 @@
+#include <GY953.h>
+#include <SPI.h>
+
+bool justStarted = true;
+unsigned long kek;
```

Comment:
wtf does this mean? 😂

Reply:
It's not even used, it's just here and in a commented out block, you can remove it (if you don't need it)

Reply:
lmao those are the remains of old time keeping code, gotta remove them

Reply:
k sks, removed this in the latest commit